README.md

```diff
@@ -96,7 +96,7 @@ to create its schema.
 A shortcut for the above steps, once S3 is set up, is available via:
 
 ```
-$ fab fresh_install:[env]
+$ fab fresh-install:[env]
```

Comment:
I think we might as well take out the hyphen here too, for consistency (something we've done on other projects).

Reply:
Sure, makes sense. Should I also remove the `fresh-install` alias from the [task](https://github.com/data-mermaid/mermaid-api/blob/master/fabfile.py#L118)?

Reply:
Sure, not needed any more I think